projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap.sh

```diff
@@ -46,6 +64,11 @@ DEFAULT_GATEWAY=$(ip r | grep default | awk '{print $3}')
 
 log::info "Using MAC: $MAC"
 log::info "Using default gateway: $DEFAULT_GATEWAY"
+
+# dhcp configuration needs 2 parameters for cp nodes, 0 parameters for worker nodes
+if [ $# -le 2 ]
```

Comment:
instead of using the number of args again, could we check if an env var is defined or not? Like, if `STATIC_IP` is not defined, do the dhcp work